tutorials/hyperspectral/RasterStack-RGB-Images-in-R-Using-HSI.Rmd

```diff
@@ -363,23 +338,6 @@ combinations are below:
 
 </div>
 
-## Plot Spectral Data on a Map
-
-We can plot the location of our image on a map of the US. For this we'll use the 
-lower left coordinates of the raster, extracted from the SPINFO group. Note that 
-these coordinates are in latitude and longitude (geographic coordinates) rather 
-than UTM coordinates.
-
-```{r create-location-map }
-
-# Create a Map showing the location of our dataset in R
-map(database="state", region="california")
-points(spInfo$LL_lat~spInfo$LL_lon,pch = 15)
-#add title to map.
-title(main="NEON San Joaquin (SJER) Field Site - Central California")
-
-```
-
 ## Raster Math - Creating NDVI and other Vegetation Indices in R
```

Comment:
Do you think we should seperate this out into its own tutorial?  It doesn't seem to fit with the title: Creating a Raster Stack from Hyperspectral Imagery in HDF5 Format in R. Plus, while short, it seems like it could stand on its own.  If so, we need to remove the last learning objective.

Reply:
Yes, I think that this is a good idea. I would like to complete this set of revisions and get these published online before parsing out this section later. Does that sound OK?